scripts/build_server.sh

```diff
@@ -0,0 +1,10 @@
+echo "Build server files..."
```

Comment:
Maybe we should prefix the file with the correct hashbang syntax `#!/usr/bin/env bash`?